com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v1_0/ApplicationsCloudModelBuilder.java

```diff
@@ -134,7 +134,7 @@ protected CloudApplicationExtended getApplication(Module module) throws SLExcept
         List<String> resolvedIdleUris = xsPlaceholderResolver.resolve(idleUris);
         List<String> customUris = new UrisClassifier(xsPlaceholderResolver).getCustomUris(deployedModule);
         List<String> fullResolvedUris = ListUtil.merge(resolvedUris, customUris);
-        List<String> services = getApplicationServices(module, true);
+        List<String> services = new ArrayList<String>(new TreeSet<String>(getApplicationServices(module, true)));

```

Comment:
It would be better to do the duplication removal directly in the getApplicationServices method.